lib/generators/USAGE

```diff
@@ -0,0 +1,125 @@
+Description:
+
+The react_on_rails:install generator combined with the example pull requests of
+generator runs will get you up and running efficiently. There's a fair bit of
+setup with integrating Webpack with Rails. Defaults for options are such that
+the default is for the flag to be off. For example, the default for -R is that
+redux is off, and the default of -b means that skip-bootstrap is off.
+
+* Redux
+
+    If you have used the --redux generator option, you will notice the familiar
+    additional redux folders. The Hello World example has also been modified to
+    use Redux.
+
+    Note the organizational paradigm of "bundles". These are like application
+    domains and are used for grouping your code into webpack bundles, in case
+    you decide to create different bundles for deployment. This is also useful
+    for separating out logical parts of your application. The concept is that
+    each bundle will have it's own Redux store. If you have code that you want
+    to reuse across bundles, including components and reducers, place them
+    under /client/app/lib.
```

Comment:
You can then import them in your client code: `import MyModule from 'lib/MyModule';` since we have configured webpack to automatically resolve `lib` to point to this folder.